src/PhpRenderer.php

```diff
@@ -221,6 +221,56 @@ public function fetch($template, array $data = []) {
         return $output;
     }
 
+    /**
+     * Renders a template without a layout and returns the result as a string
+     *
+     * cannot contain template as a key
+     *
+     * throws RuntimeException if $templatePath . $template does not exist
+     *
+     * @param $template
+     * @param array $data
+     *
+     * @return mixed
+     *
+     * @throws \InvalidArgumentException
+     * @throws \RuntimeException
+     */
+    public function fetchPartial($template, array $data = []) {
+        if (isset($data['template'])) {
+            throw new \InvalidArgumentException("Duplicate template key found");
+        }
+
+        if (!is_file($this->templatePath . $template)) {
+            throw new \RuntimeException("View cannot render `$template` because the template does not exist");
+        }
+
+
+        /*
+        foreach ($data as $k=>$val) {
+            if (in_array($k, array_keys($this->attributes))) {
+                throw new \InvalidArgumentException("Duplicate key found in data and renderer attributes. " . $k);
+            }
+        }
+        */
```

Comment:
I think it's safe to remove this one

Reply:
For sure, but that's what the original method had, so I wanted to copy it verbatim in case the maintainer wanted that left for some future implementation.